docs/docs/guide/08-loan/08-play.md

```diff
@@ -0,0 +1,302 @@
+# Play
+
+Add `10000foocoin` to Alice's account. These tokens will be used as a loan collateral.
+
+```yml
+version: 1
+accounts:
+	- name: alice
+		coins:
+			- 20000token
+			# highlight-next-line
+			- 10000foocoin
+			- 200000000stake
+	- name: bob
+		coins:
+			- 10000token
+			- 100000000stake
+client:
+	openapi:
+		path: docs/static/openapi.yml
+faucet:
+	name: bob
+	coins:
+		- 5token
+		- 100000stake
+validators:
+	- name: alice
+		bonded: 100000000stake
+```
+
+Start a blockchain node:
+
+```
+ignite chain serve
+```
+
+## Repaying a loan
+
+Request a loan of `1000token` with `100token` as a fee and `1000foocoin` as a collateral from Alice's account. The deadline is set to `500` blocks:
+
+```
+loand tx loan request-loan 1000token 100token 1000foocoin 500 --from alice
+```
+
+```
+loand q loan list-loan
+```
+
+```yml
+Loan:
+- amount: 1000token
+	borrower: cosmos153dk8qh56v4yg6e4uzrvvqjueu6d36fptlr2kw
+	collateral: 1000foocoin
+	deadline: "500"
+	fee: 100token
+	id: "0"
+	lender: ""
+	state: requested
```

Comment:
You have some extra spaces which starts at the `borrower` field, I think all the fields should have the indentation as `amount`.

Same for following yaml block codes.

Reply:
https://github.com/ignite/cli/pull/3352/commits/03e205ab726a494e69b176c07840b08fb14fd0bc